docs/reference/schemas/2.0.0.md

```diff
@@ -0,0 +1,400 @@
+# Unique schema
+
+Unique schema is a schema where all links and metadata is stored on-chain. It allows to store data and attributes flexible with separate access settings for every token field.
+This allows, for example, to make a live token, where some parts of the token can be securely changed with guarantees from the blockchain that other parts won't be changed.
+
+Tokens created previously in schemaVersion 1.0.0 are returned in the new format.
+
+[Examples how create collections tokens 2.0.0](/tutorials/createCollectionV2)
+
+### Schema 2.0.0
+
+This metadata standard describes the data structure used to describe tokens in JSON format.
+The main interface that describes the metadata for a token.
+
+```typescript
+interface TokenWithInfoV2Dto {
+    /** @example unique */
+    schemaName?: string;
+    /** @example 2.0.0 */
+    schemaVersion?: string;
+    /** @example 1.0.0 */
+    originalSchemaVersion?: string;
+    name?: string;
+    description?: string;
+    image?: string;
+    image_details?: ImageDetails;
```

Comment:
What if instead of this code block, we insert a table of contents – [[toc]] in markdown? It will help users easily navigate to interested property. But you will have to change the properties below from the list items to h3 or h4